### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by chrisghill**
There are 2 peering scenarios, and this bundle supports both:

1. The user has 2 VPCs in the same account, both provisioned by massdriver
2. The user has 2 VPCs, but they're either not both in massdriver, or in different accounts

There are 2 VPC connections (`requester` and `accepter`), but only 1 is required (`requester`). If the `accepter` connection is also hooked up, Massdriver will attempt to manage the full peering connection - request peering, accept peering, and add entries to both VPC's route tables.

If the `accepter` connection isn't hooked up, there are 2 params the user must set: Remote VPC ARN and CIDR. Using those values, we can initiate a peering connection request, and update the requestor VPC route tables, but the rest is left up to the user (accepting the connection and updating accepter VPC route tables).

This makes the bundle work 100% for 2 Massdriver VPCs, and ~50% for 1 Massdriver VPC and 1 non-Massdriver or separate account VPC.

### CODE REVIEW

File: massdriver.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      $md.immutable: true
+      $ref: https://raw.githubusercontent.com/massdriver-cloud/artifact-definitions/main/definitions/types/aws-arn.json
+    accepter_vpc_cidr:
+      type: string
+      title: Remote VPC CIDR
+      description: "**IMPORTANT: Only set this value if you haven't connected a remote \"accepter\" VPC to the bundle!!!**\n If an accepter VPC is connected, this field is ignored and the value will be extracted from the connection artifact. Use this field if the remote VPC isn't managed by Massdriver or exists in different AWS account than the requester VPC. **This will require you to [accept the peering connection](https://docs.aws.amazon.com/vpc/latest/peering/accept-vpc-peering-connection.html) and [update the route tables](https://docs.aws.amazon.com/vpc/latest/peering/vpc-peering-routing.html) of the accepter VPC manually**!"
+      $md.immutable: true
+      pattern: ^(?:10\.(?:[0-9]|[0-9]{2}|1[0-9][0-9]|2[0-4][0-9]|25[0-5])|172\.(?:1[6-9]|2[0-9]|3[0-1])|192\.168)(?:\.(?:[0-9]|[0-9]{2}|1[0-9][0-9]|2[0-4][0-9]|25[0-5])){2}(?:/(?:1[6-9]|2[0-4]))$
```
Comment (likelymichael) [inline massdriver.yaml:22]: Could we use the autocidr feature for this to make a drop-down?

File: operator.mdx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Operator Guide
```
Comment (likelymichael) [inline operator.mdx:1]: We should rename this to `operator.draft.mdx` if we don't have a guide ready for it so it's not populated in the UI.

**Review by likelymichael — APPROVED** [review]
Approved, but added a couple suggestions

**Review by xpositivityx — APPROVED** [review]

File: massdriver.yaml (hunk 1 above)
Comment (chrisghill) [reply massdriver.yaml:22]: This field isn't for creating a VPC, its for connecting to an existing VPC. This CIDR must match the CIDR of an existing, external DB.

**Pull request merged**